frontend/package.json

```diff
@@ -15,8 +15,8 @@
     "prettier": "prettier --ignore-unknown --write .",
     "preview": "vite preview",
     "test": "vitest",
-    "e2e": "playwright test",
-    "e2eui": "playwright test --ui",
+    "e2e:lib-ui": "playwright test -c playwright.lib-ui.config.ts",
+    "e2e:app": "playwright test -c playwright.app.config.ts",
```

Comment:
Don't forget to change this in the `README.md`.

Reply:
thanks! done